src/app.js

```diff
@@ -1 +1,26 @@
 'use strict';
+
+const fs = require('fs');
+const readline = require('readline');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function createFileEverySecond() {
+  setInterval(() => {
+    const date = new Date().toLocaleTimeString().split(':');
+    const name = `./src/app-${date[0]}_${date[1]}_${date[2]}.log`;
```

Comment:
You can also get hours, minutes and seconds using Date methods
https://www.w3schools.com/js/js_date_methods.asp